wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,176 @@
 
-4.  Features that are not currently in the game should avoid having their own article.
-    1.  This excludes features which have been removed and April Fool's day features, which may be noted in articles affected by them.
+GFM has a small amount of markup that **anyone** can learn it in less than a day!
+However, don't expect a lot out of Markdown as it is intended to be used as a bare minimum markup language.
 
-### Community  
+### Syntax
 
-1.  Gameplay strategies, guides, how-tos, etc., should be subarticles of [Guides](/wiki/List_of_Guides).
+If you need help with GFM syntax, see [markdown syntax - GitHub Guides](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) or [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
 
-### Wiki Rules  
+## Headings
 
-1.  Parody, comedic, nonsense, hoax, and speculation articles, or any other article that could mislead players are not allowed to be created.
-2.  Articles created for the purpose of advertising are not allowed.
-3.  Articles about fan communities are not allowed due to advertising issues.
+All articles **must have one level one heading** with the content being the article's title.
 
-Articles in the "User:" namespace are exempt from the notability guidelines. They can be used for anything, provided they follow the other Wiki Rules.
+The remaining section headings must be level 2 onwards.
 
-### Redirects
+Optionally, but preferably, add an extra line before and after the heading (level 2 and down).
+This will help make the headings stand out a bit more when editing.
 
-Redirects are exempt from the normal notability, but must redirect to an article that fits the notability guidelines. If a redirect leads to another wiki. Redirects can be created if they fit one of the following:
+You can choose either the underline style or hashtag style of level 1 and level 2 headings.
+Note that level 3 and onwards uses the hashtag style.
 
-1.  Alternate spelling of the title, such as "Combo color" for "Combo colour".
-    1.  Incorrect spelling, typos, and irregular formatting are not allowed.
+Please limit to a heading level of 5.
 
-2.  Alternate or shortened name, provided the name is common usage, such as "CtB" for "[Catch the Beat](/wiki/Game_Modes/osu!catch)".
-3.  Previous article title.
-    1.  An exception is if the previous title was not commonly used.
+## Article Titles
 
-4.  Alternate capitalisation or form, including changing the title to plural case.
-5.  A part of a merged or multi-topic article, such as a "Double Time" for [Game Modifiers](/wiki/Game_Modifiers).
+The title of an article is the name of the folder that the article.
+Replace the underscores (`_`) with spaces (` `).
+For English titles, if you need to reword the title, you must rename the folder, with rare expections.
+Translated articles of an article will be located inside the appropriate English-named folder.
 
```

Comment:
Honestly speaking, I have no idea what you are saying now.

Put an example here. Like (deliberately done badly):-

```Markdown
Article_Style_Guide (filename)

# Panduan Gaya Artikel (example in ml.md)
```